index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
 			    </div>
 			</div>
 		</section>
+
+		<section class="works">
+			<div class="works1">
+			<img class="snap" src="images/Snap3.png" alt="">
```

Comment:
- [ ] The gray background for your works section does not conform to the colors provided by Figma, kindly alter them to correspond to the Figma template.
> Your Design 
![image](https://user-images.githubusercontent.com/90556221/185599498-535d943f-0323-46a3-babc-736fc8069df4.png)

> Figma Design
![image](https://user-images.githubusercontent.com/90556221/185599599-c5715caa-39a1-44c0-a79c-d1c7e871a893.png)